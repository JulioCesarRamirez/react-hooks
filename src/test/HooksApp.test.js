import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import { HookApp } from '../HooksApp';

describe('Test <HookApp />', () => {
  it('Should show <HookApp /> correctly', () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
