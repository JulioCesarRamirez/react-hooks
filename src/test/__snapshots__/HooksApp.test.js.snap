// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <HookApp /> Should show <HookApp /> correctly 1`] = `
<div>
  <h1>
    Hello world
  </h1>
</div>
`;
